package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "@uniappca/type-validation-nodejs",
+  "name": "@yoiksoftca/type-validation-nodejs",
```

No need for "ca" as our GitHub org is @yoiksoft

```suggestion
  "name": "@yoiksoft/type-validation-nodejs",
```